tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/default/string/field_string_default_model_test.dart

```diff
@@ -0,0 +1,316 @@
+import 'package:serverpod_cli/analyzer.dart';
+import 'package:serverpod_cli/src/analyzer/code_analysis_collector.dart';
+import 'package:serverpod_cli/src/analyzer/models/stateful_analyzer.dart';
+import 'package:serverpod_cli/src/generator/code_generation_collector.dart';
+import 'package:serverpod_cli/src/test_util/builders/generator_config_builder.dart';
+import 'package:serverpod_cli/src/test_util/builders/model_source_builder.dart';
+import 'package:test/test.dart';
+
+void main() {
```

Comment:
Suggestion: Missing tests that validate that we can use unescaped `'` inside of `"` and vice versa.